### PR TITLE
Fix tfpark resume training exception

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

**Comment by yangw1234** [comment]
@jenniew would you verify this pr works for your case?

**Comment by yangw1234** [comment]
jenkins passed http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/3809/

**Comment by jenniew** [comment]
> @jenniew would you verify this pr works for your case?

I verified. It works on my case. Thank you.

File: pyzoo/test/zoo/tfpark/test_tfpark_tfoptimizer.py [hunk 1]
```diff
@@ -147,13 +147,30 @@ def test_checkpoint(self):
                         break
 
                 assert ckpt_path is not None, "Cannot fine checkpoint file"
```
Comment (jason-dai) [inline pyzoo/test/zoo/tfpark/test_tfpark_tfoptimizer.py:149]: cannot find

**Pull request merged**